vortex-duckdb-ext/src/duckdb/query_result.rs

```diff
@@ -94,6 +95,12 @@ impl QueryResult {
         T::try_from(value)
     }
 
+    pub fn cell_as_str(&self, col_idx: usize, row_idx: usize) -> VortexResult<String> {
+        self.check_bounds(col_idx, row_idx)?;
+
+        unsafe { QueryResultCell::new(self.as_ptr(), col_idx, row_idx) }.to_string()
```

Comment:
you can just call
```
result.get::<String>(1, 0)
```
no need for an extra func

Reply:
That will not cast i8 to string

Reply:
`duckdb_value_string` does but the try impl uses `duckdb_value_varchar` is that a bug?